Makefile

```diff
@@ -26,11 +27,11 @@ $(gitrepo): tests/setup-git
 	cd tests && ./setup-git
 
 clean:
-	rm -f $(objects) vcprompt
+	$(RM) $(objects) vcprompt
```

Comment:
Since the use of `$(wildcard)` already demands that the Makefile be processed using GNU Make, you might as well take advantage of the builtin `RM` variable that expands to (basically) `rm -f`